src/main/java/io/appium/java_client/touch/PositionOffsetOption.java

```diff
@@ -0,0 +1,51 @@
+package io.appium.java_client.touch;
+
+import static java.util.Optional.ofNullable;
+
+import org.openqa.selenium.Point;
+
+import java.util.Map;
+
+public class PositionOffsetOption<T extends PositionOffsetOption<T>> extends ActionOptions<T> {
+    private Point offset = new Point(0, 0);
+
+    /**
+     * It creates a built instance of {@link PositionOffsetOption} which takes x and y position
+     * offsets. This is offset from the upper left corner of the screen/element or
+     * from the corresponding action.
+     *
+     * @param xOffset is x-axis offset.
+     * @param yOffset is y-axis offset.
+     * @return a built option
+     */
+    public static PositionOffsetOption positionOffsetOption(int xOffset, int yOffset) {
+        return new PositionOffsetOption().withOffet(xOffset, yOffset);
+    }
+
+    /**
+     * Sets the offset from the upper left corner of the screen/element or
```

Comment:
I'd rather have two separate methods to set relative and absolute offset, so then we can distinguish these in _verify_ and show an error if the client wants, for example, use absolute coordinates with _moveTo_ call.